### PR TITLE
make createdPost object in item/image controller

### PR DESCRIPTION
**Pull request by hewonjeong**
- update mock data keys following createdPost key rule.
- make createdPost object in item/image controller
- test code about item/image post controller

### CODE REVIEW

File: src/server/images/controllers.js [hunk 1]
```diff
@@ -104,10 +106,18 @@ export default {
...
         });
       });
     })
+    .then((image) => {
+      const createdPost = {
+        entity: ENTITY.ITEM,
+        itemKey: image.itemKey,
+        imageKey: image.key
+      };
+      CreatedPostManager.addPost(image.userKey, createdPost, timeHash);
```
Comment (festinalente91) [inline src/server/images/controllers.js]: you'd better add return here, because it's promise :)

Comment (hewonjeong) [reply src/server/images/controllers.js]: solved! I have to get familiar with Promise yet.

File: src/test/node/items/controllers.js [hunk 2]
```diff
@@ -256,22 +257,38 @@ test('add an item to database', t => {
...
-        addedIdxItems.push(data.value);
-        if (data.value.key !== key) {
-          t.fail(`Indexing item's key is wrong : ${data.value.key}`);
-          t.end();
+      switch (entity) {
+      case ENTITY.ITEM:
+        if (isOriginKey) {
+          addedItem = data.value;
+          t.equal(data.value.address, expected.address, 'should be same address');
+          break;
```
Comment (ovekyc) [inline src/test/node/items/controllers.js]: this break is meaninglesss

File: src/test/node/items/controllers.js [hunk 3]
```diff
@@ -256,22 +257,38 @@ test('add an item to database', t => {
...
+          }
         }
-        // Case of origin image having information about image data(userKey, caption...).
-      } else if (!isItem && isOriginKey) {
-        addedImage = data.value;
+        break;
+      case ENTITY.IMAGE:
+        if (isOriginKey) {
+          addedImage = data.value;
+          break;
```
Comment (ovekyc) [inline src/test/node/items/controllers.js]: ditto

File: src/test/node/items/controllers.js (hunk 2 above)
Comment (hewonjeong) [reply src/test/node/items/controllers.js]: Oops. thx

File: src/test/node/items/controllers.js (hunk 3 above)
Comment (hewonjeong) [reply src/test/node/items/controllers.js]: ditto

**Comment by festinalente91** [comment]
LGTM

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**